client/src/views/Home/JoinParty.jsx

```diff
@@ -76,16 +85,28 @@ function JoinGame({
         >
           Join Game
         </Button>
-      </main>
-      <footer className="app__footer">
         <Button
           onClick={showCreateGameView}
-          type="secondary"
-          className="button-secondary--min-width"
+          className="button-disabled"
+          type="primary"
+          style={{ marginTop: "24px" }}
           icon="+"
         >
           Create Game
         </Button>
+      </main>
+      <footer className="app__footer">
+        <Button onClick={handleHowToPlayModalOpen} className="button-secondary">
+          How To Play
+        </Button>
+        {/* Opens the 'How to Play' Modal, which explains the game */}
+        {howToPlayOpen ? (
```

Comment:
two comments. 

1. instead of a ternary where the else condition returns null. You can just use && operator. 

```
{howToPlayOpen && <HowToPlayModal />}
```

2. These modals are hidden when isActive is false. So there is no need for this conditional rendering at all.